less/home-page.less

```diff
@@ -58,12 +96,28 @@
       }
 
       .btn {
+        .btn-mxn(10px 20px, 110px);
```

Comment:
I think you are passing 1 too many arguments. You only have 2 parameters written on your `.btn-mixin`, but here you are passing 3.